src/interval-tree.h

```diff
@@ -0,0 +1,563 @@
+/*	$NetBSD: interval-tree.h,v 1.8 2004/03/28 19:38:30 provos Exp $	*/
+/*	$OpenBSD: interval-tree.h,v 1.7 2002/10/17 21:51:54 art Exp $	*/
+/* $FreeBSD$ */
+
+/* This is a COPY of the in-tree tree.h modified to accomodate interval
+ * tree operations */
```

Comment:
I guess I would like the location and revision of the original file as well? Could be another PR though.